script/counter_inc_feePayer.ts

```diff
@@ -0,0 +1,56 @@
+import { deployments, ethers } from "hardhat";
+import { Wallet as KlaytnWallet } from "@klaytn/ethers-ext";
+import "@klaytn/hardhat-utils";
+
+async function main() {
+  const Counter = await deployments.get("Counter");
+  const counter = await ethers.getContractAt("Counter", Counter.address);
+  console.log("Using contract at:", Counter.address);
+  console.log("number before increment:", await counter.number());
+
+  const provider = new ethers.providers.JsonRpcProvider(hre.network.config.url);
+  const accounts = await hre.ethers.getWallets();
+
+  const sender = new KlaytnWallet(accounts[0].privateKey, provider);
+  const feePayer = new KlaytnWallet(accounts[1].privateKey, provider);
+
+  const fields = await counter.populateTransaction.increment();
+  // {
+  //   data: '0xd09de08a',
+  //   to: '0xa9eF4a5BfB21e92C06da23Ed79294DaB11F5A6df',
+  // }
+
+  const tx = {
+    type: 0x31,
```

Comment:
TxType.FeeDelegatedSmartContractExecution

Reply:
This can be reflected after ethers-ext is upgraded.